tools/src/py-mkopensource/main.go

```diff
@@ -41,38 +41,50 @@ func parseLicenses(name, version, license string) map[License]struct{} {
 		{"CacheControl", "0.12.6", "UNKNOWN"}:          {Apache2},
 		{"Click", "7.0", "BSD"}:                        {BSD3},
 		{"Flask", "1.0.2", "BSD"}:                      {BSD3},
-		{"GitPython", "3.1.11", "UNKNOWN"}:             {BSD3},
+		{"GitPython", "3.1.24", "BSD"}:                 {BSD3},
 		{"Jinja2", "2.10.1", "BSD"}:                    {BSD3},
 		{"chardet", "3.0.4", "LGPL"}:                   {LGPL21},
 		{"colorama", "0.4.3", "BSD"}:                   {BSD3},
-		{"decorator", "4.4.2", "new BSD License"}:      {BSD2},
-		{"gitdb", "4.0.5", "BSD License"}:              {BSD3},
+		{"decorator", "5.1.0", "new BSD License"}:      {BSD2},
+		{"gitdb", "4.0.9", "BSD License"}:              {BSD3},
 		{"idna", "2.7", "BSD-like"}:                    {BSD3, PSF, Unicode2015},
 		{"idna", "2.8", "BSD-like"}:                    {BSD3, PSF, Unicode2015},
+		{"importlib-resources", "5.4.0", "UNKNOWN"}:    {Apache2},
 		{"itsdangerous", "1.1.0", "BSD"}:               {BSD3},
-		{"jsonpatch", "1.30", "Modified BSD License"}:  {BSD3},
+		{"jsonpatch", "1.32", "Modified BSD License"}:  {BSD3},
 		{"jsonpointer", "2.0", "Modified BSD License"}: {BSD3},
 		{"jsonschema", "3.2.0", "UNKNOWN"}:             {MIT},
 		{"lockfile", "0.12.2", "UNKNOWN"}:              {MIT},
 		{"oauthlib", "3.1.0", "BSD"}:                   {BSD3},
-		{"pep517", "0.8.2", "UNKNOWN"}:                 {MIT},
-		{"pip-tools", "5.3.1", "BSD"}:                  {BSD3},
-		{"ptyprocess", "0.6.0", "UNKNOWN"}:             {ISC},
+		{"pep517", "0.12.0", ""}:                       {MIT},
+		{"pip-tools", "6.4.0", "BSD 3 Clause"}:         {BSD3},
```

Comment:
Since there's no longer an ambiguity, I'd just add `"BSD 3 Clause": {BSD3},` to the `static` table further down.